tests/test-sources/plugins/utils/neocord.nix

```diff
@@ -0,0 +1,40 @@
+{
+  empty = {
+    plugins.neocord.enable = true;
+  };
+
+  defaults = {
+    plugins.neocord = {
+      enable = true;
+
+      # General options.
+      autoUpdate = true;
+      logo = "auto";
+      logo_tooltip = null;
```

Comment:
That would be logoTooltip

Reply:
Thanks! I was looking for why this error was being caused for a while now hahaha